egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -156,5 +156,110 @@
+      }
+    ],
+    "chartType": "metric",
+    "valueType": "number",
+    "drillChart": "none",
+    "documentType": "_doc",
+    "action": "",
+    "aggregationPaths": [
+      "Total Population Covered"
+    ],
+    "insight": { },
+    "_comment": " "
+  },
+  "overviewPopulationTargeted": {
+    "chartName": "DSS_HEALTH_OVERVIEW_POPULATION_TARGETED",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "",
+        "requestQueryMap": "{\"province\": \"province\", \"district\": \"district\"}",
+        "indexName": "micro-plan-index-v1",
+        "aggrQuery": "{\n  \"aggs\": {\n    \"AGGR\": {\n      \"filters\": {\n        \"filters\": {\n          \"Campaign Filter\":{\n            \"bool\": {\n              \"must\" : [\n                {\n                  \"term\": {\n                    \"targetType\": \"INDIVIDUAL\"\n                  }\n                },\n                {\n                  \"term\": {\n                    \"productVariant\": \"LLIN\"\n                  }\n                }\n                ]\n            }\n          }\n        }\n        },\n      \"aggs\": {\n        \"Target per day\": {\n          \"sum\": {\n            \"field\": \"targetPerDay\"\n          }\n        },\n        \"Overall Target\": {\n          \"sum\": {\n            \"field\": \"overallTarget\"\n          }\n        },\n        \"Campaign Total Duartion in days\": {\n          \"sum\": {\n            \"field\": \"campaignDurationInDays\"\n          }\n        },\n        \"Target Population per Day\": {\n          \"bucket_script\": {\n            \"buckets_path\": {\n              \"tpd\": \"Target per day\"\n            },\n            \"script\": \"(params.tpd) * 365\"\n          }\n        },\n        \"Overall Population Target\": {\n          \"bucket_script\": {\n            \"buckets_path\": {\n              \"opt\": \"Overall Target\"\n            },\n            \"script\": \"(params.opt) * 365\"\n          }\n        }\n      }\n      }\n    }\n  }"
```

Comment:
@bhanuprakash-egov This query can be simplified by removing the unwanted aggregations such as the sum of overall targets. Please refer the query written for the "Households target"

Reply:
resolved